lib/carthage_cache/version.rb

```diff
@@ -1,3 +1,3 @@
 module CarthageCache
-  VERSION = "0.4.1"
+  VERSION = "0.4.2"
```

Comment:
Please revert this change because I don't know if I will release a new version as soon as this PR is merged. Also this project follows [semantic versioning](http://semver.org/), when this feature gets release the next version should increase the minor number not the patch.